pkg/cmd/server/kubernetes/master.go

```diff
@@ -76,10 +76,12 @@ func (c *MasterConfig) RunPersistentVolumeClaimBinder() {
 }
 
 func (c *MasterConfig) RunPersistentVolumeClaimRecycler(recyclerImageName string, client *client.Client) {
+	privilged := true
```

Comment:
I am confused.  What do we want to do with the service account?

Hostmount was added for the recycler (though maybe not exclusively).  Requiring the user to add Privileged to the Houstmount SCC seems logical, no?  They would explicitly opt-in to enable this feature.